### ISSUE TITLE
jenkins-plugin-cli --plugins ansicolor is not working

### ISSUE THREAD

**Issue by s-u-b-h-a-k-a-r** [jenkinsci/plugin-installation-manager-tool#585]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Paste the output here
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

MAC

### Reproduction steps

Run the command jenkins-plugin-cli --plugins ansicolor

### Expected Results

the plugin should show in /var/jenkins_home/plugins folder

### Actual Results

There was no plugin or error message it shows the message as "Done" but the plugin was not found. The same is happening with most of the plugins

**Comment by MarkEWaite** [comment]
I can't duplicate the problem.  I ran the `jenkins-plugin-cli --plugins ansicolor -d tmp` after creating a directory `tmp` in my current working directory.  It downloaded the `ansicolor` plugin as expected.  Please provide more details about your configuration.

Configurations that I checked:

* Ubuntu Linux 22.04 with Java 17.0.8
* FreeBSD 13.1 with Java 11.0.18

**Comment by s-u-b-h-a-k-a-r** [comment]
Is -d is mandatory option.. When I run this command in Dockerfile without -d the plugins are not downloaded 
RUN jenkins-plugin-cli --plugin-file /usr/share/jenkins/ref/new-plugins.txt -d /var/jenkins_home/plugins

**Comment by MarkEWaite** [comment]
I used the example from the [container documentation](https://github.com/jenkinsci/docker/blob/master/README.md#usage-1) and it downloaded the plugins into the /var/jenkins_home/plugins directory as expected.  My example Dockerfile looked like this:

```
FROM jenkins/jenkins:lts-jdk11
RUN jenkins-plugin-cli --plugins pipeline-model-definition github-branch-source
```

I ran the example from the documentation with a `plugins.txt` file in my working directory and it worked using this Dockerfile:

```
FROM jenkins/jenkins:lts-jdk11
COPY --chown=jenkins:jenkins plugins.txt /usr/share/jenkins/ref/plugins.txt
RUN jenkins-plugin-cli -f /usr/share/jenkins/ref/plugins.txt
```

Double check your syntax and confirm that you're copying the expected file into `/usr/share/jenkins/ref/new-plugins.txt`.  Alternately, try with the examples first to confirm that the examples work for you.
